### DISCUSSION TITLE
Stretched Mode - UI scaling specific to individual overlays

### DISCUSSION THREAD

**Discussion by benamaurer** [runelite/runelite#16230] [Q&A] [answered]
Wondering how difficult it would to configure individual overlays to scale separately (ex: only selected UI components to x%)? On monitors with higher resolutions, the UI scaling in stretched mode is very nice, but causes the screen to become cluttered quickly with the entire UI scaled up

**Comment by Enriath** [comment, marked as answer]
Duplicate of #6749
